test/test_index_string.cpp

```diff
@@ -1688,14 +1688,40 @@ TEST(StringIndex_Fuzzy)
                 }
             }
 
+
+            for (size_t r = 0; r < 5 + 1000 * TEST_DURATION; r++) {
```

Comment:
makes it iterate 5 times in normal unit-test mode and 5+1000*TEST_DURATION for TEST_DURATION > 0 (same pattern as we use in other tests)